Packages/com.unity.inputsystem/InputSystem/Devices/IEventMerger.cs

```diff
@@ -4,6 +4,6 @@ namespace UnityEngine.InputSystem
 {
     internal interface IEventMerger
     {
-        bool MergeForward(InputEventPtr currentEvent, InputEventPtr nextEvent);
+        bool MergeForward(InputEventPtr currentEventPtr, InputEventPtr nextEventPtr);
```

Comment:
Wondering... maybe we should just pass raw pointers here? Along with the guarantee "you're not getting NULLs".

Overall, wish a whole bunch of APIs would be using plain `InputEvent*` instead of the rather outmoded `InputEventPtr` wrapper.